airflow/providers/common/sql/provider.yaml

```diff
@@ -16,26 +16,35 @@
 # under the License.
 
 ---
-package-name: apache-airflow-providers-core-sql
-name: Core SQL
+package-name: apache-airflow-providers-common-sql
+name: Common SQL
 description: |
-    `Core SQL Provider <https://en.wikipedia.org/wiki/SQL>`__
+    `Common SQL Provider <https://en.wikipedia.org/wiki/SQL>`__
 
 versions:
   - 1.0.0
 
-dependencies:
-  - apache-airflow>=2.2.0
+dependencies: []
```

Comment:
What is the reason for removing the dependency?

Reply:
`Common-sql` is now "preinstalled" dependency of Airlfow (same as http/sqlite/imap) - this means that airlfow has it as dependency. If we add dependency here, we will create a circular dependency. This is not a problem for `pip` (it will handle it) but there are other tools (especially those analysing dependency trees) that assume "DAG" for dependency tree, and they got broken if we keep it.

Reply:
BTW. I will need to release pre-release version of the provider in `pypi` to make the buiild works :)